LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
I think the first line might be the reason GitHub does not recognize this repo as MIT licensed.
When I follow this link https://github.com/volodeyka/event-struct/tree/license, I don't see MIT license in the GH metainfo.
See the template here: https://opensource.org/licenses/MIT.

Reply:
I've tried to remove this line, but GitHub still doesn't recognize the license in the branch. 
Initially, a copy-pasted the MIT license with this line from [IMM](https://github.com/weakmemory/imm/blob/master/LICENSE.md) project, where it is correctly recognized by GitHub.
I still believe that it is because the LICENSE isn't in the master branch.

Reply:
Let's check the hypothesis :)